test/factories/users.rb

```diff
@@ -4,4 +4,10 @@
     password 'asdfasdf'
     role 'superadmin'
   end
+
+  factory :user_noprofile, class: User do
```

Comment:
I think it's better to have just a user factory and a user factory with_profile. You can use [Factory Girl Callbacks](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#callbacks)